especifico/json_schema.py

```diff
@@ -93,13 +93,22 @@ def _do_resolve(node):
                     return resolved
         elif isinstance(node, Mapping):
             for k, v in node.items():
-                node[k] = _do_resolve(v)
+                # do not resolve refs in jsonschema nested in the spec
+                # jsonschema library can resolve them by itself
+                if (
+                        (k == "components" and not path)
+                        or (k == "schema" and path[-2:] == ["content", "application/json"])
+                ):
+                    resolved = v
+                else:
+                    resolved = _do_resolve(v, path + [k])
+                node[k] = resolved
         elif isinstance(node, (list, tuple)):
             for i, _ in enumerate(node):
-                node[i] = _do_resolve(node[i])
+                node[i] = _do_resolve(node[i], path + ["[]"])
```

Comment:
why do we do `"[]"`? is it enough to just `""`?

Reply:
yes [i] it's better